docs/rules/indent.md

```diff
@@ -68,6 +68,7 @@ if (a) {
 
 This rule has an object option:
 
+* `"ignoredNodes"` can be used to disable checking for any option below by using an array of [selectors](/docs/developer-guide/selectors.md). If an AST node is matched by any of the selectors, the indentation of tokens which are direct children of that node will be ignored.
```

Comment:
This isn't really related to other options.

It can disable indentation for any nodes, e.g., for `UpdateExpression`, which doesn't have a specific option:

```js
/* eslint indent: ["error", 4, { ignoredNodes: ["UpdateExpression"] }] */

a;

 ++b; // ok

c;
   
```

[Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IGluZGVudDogW1wiZXJyb3JcIiwgNCwgeyBpZ25vcmVkTm9kZXM6IFtcIlVwZGF0ZUV4cHJlc3Npb25cIl0gfV0gKi9cblxuYTtcblxuICsrYjsgLy8gb2tcblxuYztcbiAgICIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6MTIsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnsibm8tbXVsdGktYXNzaWduIjoyfSwiZW52Ijp7fX19)

Reply:
I see, thanks for explaining. I refactored my commit, but I still think there can be an improvement regarding explaining what AST is to a new user. I could find a good documentation page to give link to. Any ideas? Do you think it is needed?

Reply:
If you'd like, we could add more details to the [ignoredNodes section](https://eslint.org/docs/rules/indent#ignorednodes), so that the short description in the overview of all options doesn't become too long. Maybe something similar to [no-multi-spaces#exceptions](https://eslint.org/docs/rules/no-multi-spaces#exceptions).

Reply:
Good idea. I'll see if I can add AST description there. Btw, I wonder if you thought putting a link on short descriptions of options (maybe on the option name itself) to long descriptions of each option. That way, I think the long descriptions would be easier to spot and navigate to.

Reply:
I added a description about ast nodes at the end of long description of the option in commit https://github.com/eslint/eslint/pull/14168/commits/843df45c8f6a6d9b144f6de433d223e2c2a70d39

Reply:
> Btw, I wonder if you thought putting a link on short descriptions of options (maybe on the option name itself) to long descriptions of each option. That way, I think the long descriptions would be easier to spot and navigate to.

Makes sense to evaluate this, but since we are not doing that in other documents, let's keep it as is for now.